src/systems/eu/implementations/react/packages/language-list/LanguageList.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LanguageListItem from './LanguageListItem';
+
+const LanguageList = ({ items }) => {
+  const column1 = items.slice(0, Math.ceil(items.length / 2));
+  const column2 = items.slice(Math.ceil(items.length / 2));
+
+  return (
+    <div className="ecl-row">
+      <div className="ecl-language-list__column ecl-col-12 ecl-col-sm-4 ecl-offset-sm-2">
+        <ul className="ecl-language-list__list">
+          {column1.map(item => (
+            <li
+              className={`ecl-language-list__item ${
+                item.isActive ? 'ecl-language-list__item--is-active' : ''
+              }`}
```

Comment:
I guess React is complaining about a missing `key`, isn't it?

Reply:
No, it isn't

Reply:
yes, it is ;)

![image](https://user-images.githubusercontent.com/3524389/46005783-51937800-c0b6-11e8-8aaa-eee1fa20e60c.png)